docs/source/compression/compression_evaluator.rst

```diff
@@ -0,0 +1,231 @@
+Compression Evaluator Creation
+==============================
+
+The evaluator is used to package the training and evaluation process for models with similar topologies.
+To explain why NNI needs an evaluator, let's first look at the general process of model compression in NNI.
+
+In model pruning, some algorithms need to prune according to some intermediate variables (gradients, activations, etc.) generated during the training process,
+and some algorithms need to gradually increase or adjust the sparsity of different layers during the training process,
+or adjust the pruning strategy according to the performance changes of the model during the pruning process.
+
+In model quantization, NNI has quantization-aware training algorithm,
+it can adjust the scale and zero point required for model quantization from time to time during the training process,
```

Comment:
What's the point here? What's the relationship between "algorithm" and evaluator?

Reply:
to explain why we need training / prediction function in compression, do you mean this part might be confusing to users?

Reply:
I didn't see how this part is correlated to evaluator.

Reply:
add
```
In order to better support the above algorithms' need and maintain the consistency of the interface,
NNI introduces the ``Evaluator`` as the carrier of the training and evaluation process.
```